creator-node/src/services/SessionExpirationQueue.js

```diff
@@ -0,0 +1,110 @@
+const Bull = require('bull')
+const Sequelize = require('sequelize')
+const sessionManager = require('../sessionManager')
```

Comment:
have sessionManager call delete function from dbManager - dbManager should be in charge of delete sessions in bulk.

Reply:
implement `deleteSesssions` - with db and redis, and if one fails then roll back.